### ISSUE TITLE
Replace html5Blank with Yoursite

### ISSUE THREAD

**Issue by ottz0** [html5blank/html5blank#228]
What is the best way to replace html5Blank with "Your Site"?

One big fnd and replace in the code and the database should do it? Or is there another solution? Or don't worry about it?

**Comment by doughballs** [comment]
Yes. This is a great question. It's a little confusing sometimes, take CPT for instance - we have both `html5-blank` and `html5blank` passed as the second argument, and then elsewhere there's `html5_` prepended to function names.

It would be really handy to state best practice on changing this to a different namesapce.
